CMakeLists.txt

```diff
@@ -2,7 +2,15 @@ cmake_minimum_required(VERSION 2.8)
 
 set(PROJECT_NAME mc-mini)
 set(PROJECT_VERSION 1.1.0)
-project(${PROJECT_NAME})
+
+if(CMAKE_VERSION LESS 3.0)
```

Comment:
You were right: everything works with what you suggested! I think I did this before, but that commit didn't pass Travis CI because ```<stdexcept>``` headers were missing for ```std::runtime_error``` and ```std::invalid_argument```. [This] (https://github.com/icherkashin/mc-mini/commit/c2ad6e0d3acc643e1919c8c0e43a51e35a508df2) commit implements your suggestions.